ci-operator/step-registry/openshift/e2e/openstack/ipi/openshift-e2e-openstack-ipi-workflow.yaml

```diff
@@ -9,7 +9,6 @@ workflow:
     - chain: ipi-openstack-post
     env:
       OPENSTACK_EXTERNAL_NETWORK: "external"
```

Comment:
Above you set cluster-type to openstack-vexxhost, making the conf step to pull Vexxhost’s clouds.yaml. However here you still set the MOC external network and flavor names.

IIUC, with this patch, to move a job across clouds you must simultaneously change cluster-type in config, AND network and flavor names in the workflow.